packages/utils/src/lib/id.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * The code in this file is adapted from `nanoid`: https://github.com/ai/nanoid
+ *
+ * `nanoid` is currently only distributed as an ES module. some tools (jest, playwright) don't
+ * properly support ESM-only code yet, and tldraw itself is distributed as both an ES module and a
+ * CommonJS module. By including nanoid here, we can make sure it works well in every environment
+ * where tldraw is used. We can also remove some unused features like custom alphabets.
+ *
+ * @license
+ * The MIT License (MIT)
+ *
+ * Copyright 2017 Andrey Sitnik <andrey@sitnik.ru>
```

Comment:
it's sufficient to just link to the license file, you don't have to re-include the MIT license every time. so,

```
/*!
 * MIT License
 * Modified code originally from <https://github.com/ai/nanoid>
 * Copyright 2017 Andrey Sitnik <andrey@sitnik.ru>
 */
 ```